rb/spec/integration/selenium/webdriver/fedcm/dialog_spec.rb

```diff
@@ -0,0 +1,98 @@
+require_relative '../spec_helper'
+
+module Selenium
+  module WebDriver
+    module FedCM
+      describe Dialog, exclusive: {browser: :chrome} do
```

Comment:
Let's switch from `exclusive` to `only` - this will ensure tests are run in other browsers but expected to fail.